app/lib/document_navigation.rb

```diff
@@ -32,7 +32,8 @@ class DocumentNavigation
     "Requested" => Documents::RequestedDocumentsController,
   }.freeze
   BEFORE_CONTROLLERS = [
-    Documents::IntroController
+    Documents::IntroController,
+    Documents::IdGuidanceController,
```

Comment:
makes me nervous adding to this pattern 😬

Reply:
Agreed. It's not ideal. This would be good to discuss together.